localstack/services/stepfunctions/asl/component/state/state_execution/state_task/service/state_task_service_dynamodb.py

```diff
@@ -124,8 +123,7 @@ def _from_error(self, env: Environment, ex: Exception) -> FailureEvent:
 
     def _eval_service_task(self, env: Environment, parameters: dict) -> None:
         api_action = camel_to_snake_case(self.resource.api_action)
-
-        dynamodb_client = connect_to(config=Config(parameter_validation=False)).dynamodb
+        dynamodb_client = get_boto_client(env, "dynamodb")
```

Comment:
Is this still ensuring the parameters validation is off?

Reply:
Yes! Please make sure you use this helper to get the Boto client and not `connect_externally_to`

https://github.com/localstack/localstack/blob/aad9b80492fdc9f7b22f48af0184cb5febf6dd23/localstack/services/stepfunctions/backend/utils.py#L7-L13